homeassistant/components/environment_canada/sensor.py

```diff
@@ -77,9 +43,8 @@ def validate_station(station):
 
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)):
-        vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
-    vol.Optional(CONF_NAME): cv.string,
+    vol.Required(CONF_LANGUAGE, default='english'):
```

Comment:
Are we ok with this config option?

Reply:
I'd be happy to pull this value from the front end configuration, if that's possible.

Reply:
On sensor, yes

Reply:
But yeah, we should add a core language config flag